src/main/java/com/conveyal/datatools/manager/jobs/ProcessSingleFeedJob.java

```diff
@@ -48,21 +65,102 @@ public String getFeedSourceId () {
     @Override
     public void jobLogic () {
         LOG.info("Processing feed for {}", feedVersion.id);
+        feedSource = feedVersion.parentFeedSource();
```

Comment:
Field `feedSource` should be initialized in the constructor.

Reply:
Also, with `feedSource` initialized in the constructor, method `getFeedSourceId()` can also use it.